eblob-bf.spec

```diff
@@ -64,6 +64,18 @@ appending records one after another.
 This package contains libraries, header files and developer documentation
 needed for developing software which uses the eblob library.
 
+%package python
```

Comment:
Directory named as "bindings". I think this is right. Each bindings has a package.